UniversalScannerService/appsettings.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Log files are captured via DebugView for Windows, I am not planning to use another logging system.
I have the goal to keep the tool in one single file and without extra files such as configuration file or library